src/ICS26Router.sol

```diff
@@ -24,13 +27,17 @@ import { Multicall } from "@openzeppelin/utils/Multicall.sol";
 contract ICS26Router is IICS26Router, IICS26RouterErrors, Ownable, ReentrancyGuardTransient, Multicall {
     /// @dev portId => IBC Application contract
     mapping(string portId => IIBCApp app) private apps;
+
     /// @inheritdoc IICS26Router
     IICS02Client public immutable ICS02_CLIENT;
+    /// @dev ICS04Channel contract
+    IICS04Channel public immutable ICS04_CHANNEL;
```

Comment:
These two should be combined to a single variable imo. Idk if we can combine the interfaces but we can call it an address instead and use it with `IICS04Channel(IBC_CORE)` when we need it as ics04channel and `IICS02Client(IBC_CORE)` when we need it as ics02client

Reply:
Could we just have a `IICSCore` that simply combines `IICS02Client` and `IICS04Channel`?

Reply:
I've addressed this issue in my recent pushes